### DISCUSSION TITLE
eZ Navi-Delete Hardware Availability

### DISCUSSION THREAD

**Discussion by radensb** [radensb/eZ_Navi-Delete#9] [General]
Interest in the project is growing and I have been getting more and more questions about obtaining an eZ Navi-Delete board, so I am starting this discussion page to more efficiently address questions.

**Due to the global chip shortage that was a result from COVID and the supply chain chaos, several IC components that the eZ Navi-Delete design uses are still unavailable.** I have been in periotic contact with my assembler checking in on the availability of the components, as well as trying to source them myself to send to them so that more boards can be built. Unfortunately, I have not been able to find the required or compatible parts. I even considered changing the design to use different parts, but even the suitable substitutes are unavailable. It is a tricky situation because counterfeit parts are everywhere and need to be avoided at al costs. Even thought the AC control system is not "critical", I can see counterfeit parts causing all kinds of issues that would result in improper operation. In addition to counterfeit issues, availability has also caused price increases in the hard to find components from brokers trying to make a buck off the situation. 

To say the least - **this is extremely frustrating**. When this design was being developed, the needed components were abundant and only active components with consistent inventory in the major distributers were selected- but who saw this coming?

I appreciate the patience that has been shown. Please post any questions here vs sending an email so that others who are interested can see the response. I hope to hear back from the assembler some time next week and will update with the status.

Thank you!

**Comment by slastrina** [comment]
Understandable, i can imagine it would be extremely frustrating for you, this pandemic has interrupted many great projects.

May i ask, which components are currently causing drama's the CAN transceivers?

**Comment by slastrina** [comment]
oh wow, 8 month lead time on the MCP2515T-E/ST

**Comment by radensb** [comment]
The main components that I we have not been able to source are the CAN related IC's, the power switcher IC, and the Bluetooth module. Specifically, CAN controller and transceiver - the BT module is only for advanced, undeveloped features). I have a feeling that we are competing with the automotive industry on the CAN stuff. The MCU is also tricky to find, but they are out there (usually at x3 the $$$...). I tend not to believe the super high lead times and have found that distributors will just guess (usually on the long side too) if they dont have solid dated from suppliers. I have ordered parts that had 22 week lead times that ended up shipping a week or two later. I think the manufactures are trying to limit distributor inventory to better control supply and prevent brokers from purchasing the bulk of what the can produce. So, we am keeping an eye on it and trying to monitor availability. I requested an update from the assembler - still waiting for the reply.

**Comment by radensb** [comment]
I was updated from my supplier. Below are a list of components that are holding things up. These components either don't have sutable alternatives, or their alternatives are still unavailable. Microchip products seem to have the longest lead times from suppliers. I have requested restocking notifications from local suppliers and they assure me that they are keeping an eye out as well.

1. MCP2515T-E/ST
2. MCP2562T-H/SN
3. NCP718BSN500T1G
4. RN4871-I
5. STM32F103C8T6
6. TPS560430YFQDBVRQ1

Still working it!

**Comment by slastrina** [comment]
yeh thats a pain, im seeing alot of the same when navigating element14, fingers crossed supply chain gets resolved before we all start resorting to desoldering other boards for chips haha

**Comment by kittji** [comment]
Hey, I'm pretty sure I know the answer but are there any updates on this project? Just wanted to check since this thread was updated a couple months ago

**Reply by radensb** [reply]
Hi kittji,

I just contacted the assembler to get a parts update over the weekend. It looks like some of the lead times for the parts I haven't been able to get are decreasing, which is a great sign. I have missed a couple of opportunities to buy some of the parts when I get notifications because they get sold so quickly, but now I see that they are listed as backorder-able with expected shipping dates ranging from July to Sept. When I get the update on part availability from the assembler, I will try to get on the backorder lists for the ones they dont have access too. It seems like things are starting to ease up as previous lead-times were over a year vs a few months. I should know more within the week!

**Reply by kittji** [reply]
Oh that's great to hear! Thanks for the fast reply too!

**Comment by radensb** [comment]
UPDATE: 6/9/22
I have gotten the response from the assembler regarding the components that are still unavailable. They are holding the order until they can get the parts, or I can send them. I have finally been able to place backorders for almost all the parts, with expected deliveries between July and Sept of this year. The parts that are still unavailable with no lead times are the Bluetooth module, and the CAN transceiver. I am considering deprecating the Bluetooth due to availability, however, the module could still be added to the board in the future. I am also researching alternatives to the CAN transceiver in the event that becomes the only part that remains unavailable.

I have already started getting parts shipped! I am hopeful that the lead times for the other parts will also be overestimates, or at least show up around the estimated time. With the hard to get parts in hand, I can send a care package to the assembler to complete the materials list and get this party started.

**Comment by radensb** [comment]
Parts have been coming in!

My backorders have actually been getting fulfilled and shipped reasonably close to the expected ETA's. The last few parts are expected by end of October, at which point I will be able to provide an update to the assembler and send them the parts. My goal is to get 60 eZ Navi-Delete v2 boards built for the revised Tindie listing as well as introduce the new wiring harness kits.

**Reply by kittji** [reply]
Any updates yet?

**Reply by radensb** [reply]
> Any updates yet?

Hopefully next week. I'm just waiting on the confirmation on one more backorder part. Then I will be able to get a new quote started with my supplemental parts. That will probably take another week. I'm trying to target a mid December order timeline for 50 or 55 boards!

**Comment by radensb** [comment]
**Update 1/9/23::**
Pre-production PCB's are in! I ran into some delays getting these in. I plan on assembling one this weekend to vet changes I made to the design. Once I know it all checks out, I can start the production process and get the bulk order quoted.

**Comment by radensb** [comment]
**NOTICE 02/05/2023:**
Production has started! With a combined effort with the assembler, we have sourced all the components needed to build this project with minimal increase to cost ($15 per unit). A total of 75 units have been ordered. At this time, delivery is expected in early March. This page will be updated over that time to reflect the current version and options. Stock will be updated when product is ready to ship.

**Comment by Davesepicrage** [comment]
Hey dude, looking to get the premade board delivered to the UK however it keeps saying it can't deliver to the UK. Any chance I could get a 1 off sent to the UK and pay delivery?

**Reply by DigitalWorkz** [reply]
Yeah, I had to turn off international shipping for the time being. Im convinced that I'm not doing it right and heard that it could cause me trouble down the road. I'm looking into getting this squared away, but Tindie is no help. Getting this resolved is on my list though!
